v2/cmd/asoctl/pkg/importresources/resource_importer_report.go

```diff
@@ -47,21 +48,26 @@ func newResourceImportReport() *resourceImportReport {
 }
 
 // AddSuccessfulImport adds a successful import to the report
-func (r *resourceImportReport) AddSuccessfulImport(imported ImportedResource) {
+func (r *resourceImportReport) AddSuccessfulImport(
```

Comment:
The report is used to collate totals, so it doesn't need the full resource, just its GroupKind.